manifests/charts/istiocoredns/templates/configmap.yaml

```diff
@@ -11,20 +11,19 @@ data:
     .:53 {
           errors
           health
-          {{ if eq -1 (semver  .Values.istiocoredns.coreDNSTag  | (semver "1.4.0").Compare) }}
-          # Removed support for the proxy plugin: https://coredns.io/2019/03/03/coredns-1.4.0-release/
-          grpc global 127.0.0.1:8053
+
           forward . /etc/resolv.conf {
-            except global
+            except {{- range $.Values.global.podDNSSearchNamespaces }}{{ . }} {{- end }}
```

Comment:
docs state this is a space separate list, lgtm.

Reply:
is this even right? doesn't the dns search namespace include the kubernetes namespace where coredns is running?

Reply:
oh ignore..

Reply:
yes that will ignore the custom FQDN